[RFC] Rust-embedded-community as child organization (#504)
rust-embedded/wg: text/0000-rec-sub-org.md

- Feature Name: rust-embedded-community as child organization
- Start Date: 2020-09-07
- RFC PR:
- Rust Issue:

# Summary
[summary]: #summary

Adopt the [Rust Embedded Community (REC)][rec] as an official child organization/subproject of the Rust Embedded Working Group (REWG).

# Motivation
[motivation]: #motivation

## Maturity segmentation

> Foundational libraries and tooling for Rust usage in embedded context are part of the official REWG organization. As such, these libraries can be seen as especially well maintained or thought-through within the broad embedded rust community.

Reviewer: ```suggestion
Foundational libraries and tooling for Rust usage in embedded context are part of the official REWG organization. As such, these libraries can be seen as especially well maintained or thought-through within the broad embedded Rust community.
```

It would be interesting to have a "playground" for less mature libraries where its status is clear to users while at the same time centralizing efforts.

Users have already shown interest in something like this. See: [rust-embedded-community/meta#5][rec-5], [rust-embedded-community/meta#6][rec-6]

## Home for unmaintained projects

There are libraries that were once developed but the original author does not have the time necessary for further development or has moved on to other projects.

> The REC provides a place for these libraries to live on, offering the opportunity for development to be easily aided by or taken over by new members. e.g. Bus factor increase.

Reviewer: ```suggestion
The REC provides a place for these libraries to live on, offering the opportunity for development to be easily aided by or taken over by a group of interested people, also to increase the "bus factor".
```
